helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -91,6 +91,22 @@ spec:
         cancel_if_outside_working_hours: "true"
         severity: page
         topic: releng
+    - alert: ClusterCapiAppFailed
+      annotations:
+        description: '{{`Cluster CAPI App {{ if $labels.exported_namespace }}{{ $labels.exported_namespace }}{{ else }}{{ $labels.namespace }}{{ end }}/{{ $labels.name }}, version {{ $labels.version }} is in {{ $labels.status }} state.`}}'
+        opsrecipe: app-failed/
+      # we have this Frankenstein for namespace checks because go (and prometheus) don't support negative lookaheads in regex
+      expr: label_replace(app_operator_app_info{app=~"cluster-.+", status!~"(?i:(deployed|cordoned|not-installed))", catalog="cluster", team!~"^$|noteam"}, "cluster_id", "$1", "namespace", {{ include "namespaceNotGiantswarm" . }}) == 1
```

Comment:
The cluster-id is not good here because you will only configure the org-namespace.

I fixed it here https://github.com/giantswarm/prometheus-rules/pull/718/files

Also, do we need an extra alert for this or would it not make sense to add a regex over the catalog in the alert I fixed in my PR?

Reply:
from the discussion on slack (https://gigantic.slack.com/archives/C01176DKNP4/p1682502034772099)  the reason to have an extra alert is that 

* the `cluster-XXX` app is used to create both the MC and WC clusters so it belonged in case only as part of the `ManagementClusterAppFailed` 
* but if i add the `cluster` catalog to the regex of the `ManagementClusterAppFailed` it will also match on the `default-apps-xxxx` which instead belongs **both** to the WorkloadClusters and the ManagementCluster 

So the options where
* merge all the `AppFailed` rules together
* Create a separate alert for Cluster Api APP 

But i am ok with either really :+1:

Reply:
> but if i add the cluster catalog to the regex of the ManagementClusterAppFailed it will also match on the default-apps-xxxx which instead belongs both to the WorkloadClusters and the ManagementCluster

Also just realized i forgot to add `default-apps-.+` to the list

Reply:
I was just thinking that we do not need a distinction between management cluster apps and workload cluster apps because the same team manages both :D

Reply:
for me your #718 is fine , i am gonna close this pr in favour of that one :+1:

Reply:
Is there any other app in the cluster catalogs? Maybe you do not need to app name disctinction